sections/problem-analysis/scalability.tex

```diff
@@ -25,14 +26,16 @@ \subsection{Designing a system for scalability}
     This allows for more availability and performance, thus making the system easier to scale, but adds to the operational cost.
     \item Scale the system horizontally instead of vertically.
     Add new servers to the system instead of buying a new server with a larger capacity each time to replace the old one when more capacity is needed. 
-    In theory, this should also be cheaper since the old servers will still be in use. However, for a system with multiple servers to work, some load balancing is required.
+    In theory, this should also be cheaper since the old servers will still be in use. 
+    However, distributing network traffic properly is required for a system with multiple servers to work.
     \item Build the API first to avoid tailoring it to a specific type of frontend. 
-    The API should work as a service so that it can be used for any system such as a smartphone application or a web site. 
-    \item Caching the data that is being accessed can have significant performance benefits since the system will not have to recompute the results that are being requested often. 
+    The API should work as a service such that it can be used for any system such as a smartphone application or a web site. 
+    \item Caching the data that is being accessed can have significant performance benefits since the system will not have to recompute the results that are often requested. 
     \item As mentioned in scalability in availability, the system does not need to provide the newest data available at all times if the users do not need to know. 
     Relying on eventual consistency will lead to better availability, especially if the system has a large userbase.
     \item The system should be designed to make maintenance and automation easy so that after the software has been released, it can be monitored and the different parts can be updated when needed.
-    \item Write code to be able to run asynchronously rather than synchronously to allow tasks to be executed on different threads. This will increase the availability of the system.
+    \item Write code to be able to run asynchronously rather than synchronously to allow tasks to be executed on different threads.
```

Comment:
```suggestion
    \item Write code to be able to run asynchronously rather than synchronously to allow tasks to be executed on different threads. This will increase the availability of the system.
```